repose-aggregator/components/filters/saml-policy-translation-filter/src/test/scala/org/openrepose/filters/samlpolicy/SamlPolicyTranslationFilterTest.scala

```diff
@@ -754,37 +1147,37 @@ class SamlPolicyTranslationFilterTest extends FunSpec with BeforeAndAfterEach wi
     }
 
     it("should initialize the cache when given a config") {
-      ReflectionTestUtils.getField(filter, "cache") shouldBe null
+      ReflectionTestUtils.getField(filter, "org$openrepose$filters$samlpolicy$SamlPolicyTranslationFilter$$policyCache") shouldBe null
```

Comment:
Huh. The more you know. Well that's kind of annoying, but it should be consistent, at least. I certainly wouldn't want to start using Java branching code just because of the implementation details of the Scala compiler.